packages/page-calendar/src/DayItem.tsx

```diff
@@ -154,13 +157,18 @@ function DayItem ({ className, item: { date, info, type } }: Props): React.React
     [info, t, type]
   );
 
+  const exportCalClassNames = showAllEvents ? 'exportCal exportCal-allEvents' : 'exportCal';
+
   return (
     <div className={className}>
+      {showAllEvents &&
+        <div className='itemDate'>{date.toLocaleString('en-EN', options)}</div>
+      }
       <div className='itemTime'>{date.toLocaleTimeString().split(':').slice(0, 2).join(':')}</div>
       {desc}
       {date && (
         <Button
-          className='exportCal'
+          className={exportCalClassNames}
```

Comment:
Not sure we really need a variable for this, it is used once.

Reply:
Based on the "view" the position of the calendar icon must be re-positioned. The DayItem component is reused in the two views (in order to reduce duplicated code and reuse translations) so based on view either className 'exportCal' is used or both 'exportCal exportCal-allEvents' where the exportCal-allEvents is overwriting the right position of the calendar icon. Any proposal that would make it better?

Reply:
Agreed. Al I am suggesting is that you can do the bar assignment inside className={...}

Since the car is only used once inside the function, we probably don’t need to make an assignment for it. 

However, if it aids readability in your view, I don’t have a strong opinion on it. (It is minor)

Reply:
Agreed - Its a minor. I'll transfer the assignment inside className as you suggest :)